5-Access-Authentication-and-Authorization/5.3-Configure-SSH-Server.bats

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bats
+
+@test "5.3.1 Ensure permissions on /etc/ssh/sshd_config are configured (Automated)" {
+    run bash -c "stat /etc/ssh/sshd_config"
+    [ "$status" -eq 0 ]
+    [[ "$output" = *"Uid:"*"("*"0/"*"root"* ]]
+    [[ "$output" = *"Gid:"*"("*"0/"*"root"* ]]
+    [[ "$output" = *"Access:"*"(0"[0-7]"00"* ]]
+}
+
+@test "5.3.2 Ensure permissions on SSH private host key files are configured (Automated)" {
+    local SSH_CONFIG=$(find /etc/ssh -xdev -type f -name 'ssh_host_*_key' -exec stat {} \; | grep "Access" | grep "Uid")
+    while IFS= read -r line; do
+        [[ "$line" = *"Uid:"*"("*"0/"*"root"* ]]
+        [[ "$line" = *"Gid:"*"("*"0/"*"root"* ]]
+        [[ "$line" = "Access:"*"(0"[0-7]"00"* ]]
+    done <<< "$SSH_CONFIG"
+}
+
+@test "5.3.3 Ensure permissions on SSH public host key files are configured (Automated)" {
+    local SSH_CONFIG=$(find /etc/ssh -xdev -type f -name 'ssh_host_*_key.pub' -exec stat {} \; | grep "Access" | grep "Uid")
+    while IFS= read -r line; do
+        [[ "$line" = "Access:"*"(0"[0-7][0\|4][0\|4]* ]]
+    done <<< "$SSH_CONFIG"
+}
+
+@test "5.3.4 Ensure SSH access is limited (Automated)" {
+    run bash -c "sshd -T | grep allowusers"
+    ALLOWUSERS=$status
+    run bash -c "sshd -T | grep allowgroups"
+    ALLOWGROUPS=$status
+    run bash -c "sshd -T | grep denyusers"
+    DENYUSERS=$status
+    run bash -c "sshd -T | grep denygroups"
+    DENYGROUPS=$status
+    [ "$ALLOWUSERS" -eq 0 ] || [ "$ALLOWGROUPS" -eq 0 ] || [ "$DENYUSERS" -eq 0 ] || [ "$DENYGROUPS" -eq 0 ]
+}
+
+@test "5.3.5 Ensure SSH LogLevel is appropriate (Automated)" {
+    run bash -c "sshd -T | grep loglevel"
+    [ "$status" -eq 0 ]
+    [[ "$output" == "LogLevel VERBOSE" ]] || [[ "$output" == "loglevel INFO" ]]
+}
```

Comment:
The second check from the CIS document is missing (`grep -is 'loglevel' /etc/ssh/sshd_config /etc/ssh/sshd_config.d/*.conf |
grep -Evi '(VERBOSE|INFO)'`)